### ISSUE TITLE
Deletion of multiple files is slow

### ISSUE THREAD

**Issue by RiederAlex** [fman-community/fman#615]
I regulary need to delete lots of files inside a folder without actually deleting the folder.
This operation is much slower for both permanent deletion & move to recycle bin as compared to OS native functionality.

For example, select all files in a folder with 60 empty .txt files and delete them.
Using fman, this takes around 2.5 seconds, using native windows explorer it happens instantly.

My best guess is that this happens because it re-renders between each individual file deletion.

**Comment by joshas** [comment]
Looking at #616 it seems that fman deletes each file individually.

**Comment by jardous** [comment]
yep - this is quite annoying behavior. Deleting or moving files should be way faster.

**Comment by rasmuse** [comment]
+1

fman is becoming more useful all the time, but this is a major showstopper for me. I am just now waiting for the removal of a few thousand files selected with a regex (there is a nice plugin for that). I know I can do something like `find -delete -regex "./\d-0-.*"` etc but this always confuses me and I am afraid to mess up badly when I do such things. Therefore I love to have fman's visual feedback to guide me.

Quickly selecting, moving, deleting, and renaming long lists of files is hands down the top use case for me in fman. But currently it is so slow that I often turn to the scary `find -regex ...` solution instead.

**Comment by raguay** [comment]
If your on a macOS system, add my plugin
https://github.com/raguay/FlexibleTrash. Then, quit finder and fman will
directly delete the files and not move them to trash. The slowdown is two
fold, updating the UI and moving files to Finder's trash folder. Since I
always kill finder on my system using a Keyboard Maestro workflow, fman
will use the Flexible Trash to immediately remove the files is much faster.

Beware: removing files this way is a no return route. The files are gone!

**Comment by fdb** [comment]
Here's a video showing the deletion speed:

https://enigmeta.s3.amazonaws.com/github-issues/2019-10-21-fman-slow-deletion.mp4

 To compare, in the Finder it takes about 1 second to delete 1500 files. In the Terminal it's near instant.

**Comment by StevenLu2004** [comment]
So what if fman update the UI not for each file but say, for each 0.5 seconds?
